ui/app/mainui/AppMain.qml

```diff
@@ -899,6 +894,34 @@ Item {
         }
     }
 
+    ListView {
+        id: toastArea
+        anchors.top: parent.top
+        anchors.topMargin: 60
+        anchors.right: parent.right
+        anchors.rightMargin: 8
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: 60
+        spacing: 8
+        verticalLayoutDirection: ListView.BottomToTop
+        model: appMain.rootStore.mainModuleInst.ephemeralNotificationModel
+        delegate: StatusToastMessage {
+            primaryText: model.title
+            secondaryText: model.subTitle
+            icon.name: model.icon
+            loading: model.loading
+            type: model.ephNotifType
+            linkUrl: model.url
+            duration: model.durationInMs
+            onLinkActivated: {
+                Qt.openUrlExternally(link);
```

Comment:
Should we use `Global.openLink` instead, to open the link in Desktop Browser if user chooses to?

Reply:
@richard-ramos it's not a problem at all to update that, but didn't we agree that we should remove that popup up and open all links in the default OS browser (I think we discussed that last week when we were testing the app during offsite)?

Reply:
oh ok!  I missed that discussion 
Ignore the comment then :man_dancing: